explorer/files/anvil-catalog/builld-consortia.ts

```diff
@@ -0,0 +1,85 @@
+import {
```

Comment:
Refactor file name from `builld-consortia` to `build-consortia`.

Reply:
sb fixed. Thanks, @frano-m  D